legal/satzung.md

```diff
@@ -0,0 +1,123 @@
+3. Näheres regelt die Beitragsordnung.
+
+
+## Mitgliederversammlung
+
+1. Oberstes Organ des Vereins ist die Mitgliederversammlung.
+    Ihrer Beschlussfassung unterliegen alle in dieser Satzung oder per Gesetz vorgesehenen Gegenstände,
+    insbesondere
+    1. Wahl des Vorstandes,
+    2. Wahl der Finanzprüfer*innen,
+    3. Beschlussfassung über den Jahresbericht des Vorstandes,
+    4. Entlastung des Vorstandes,
+    6. Verabschiedung der Beitragsordnung,
+    8. Satzungsänderungen und
+    9. Auflösung des Vereins.
+2. Einmal im Geschäftsjahr soll eine ordentliche Mitgliederversammlung stattfinden.
+3. Mitgliederversammlungen werden vom Vorstand unter Einhaltung einer Frist von vierzehn Tagen unter Vorschlag einer Tagesordnung einberufen.
+4. Leitung und Vorsitz obliegt dem Vorstandsvorsitzenden, bei dessen Abwesenheit einem anderen Vorstandsmitglied.
+    Sollte kein Vorstandsmitlgied anwesend sein, bestimmt die Mitgliederversammlung eine Versamlungsleitung.
+    Bei Wahlen kann die Versamlungsleitung für die Dauer des Wahlvorgangs und der vorhergehenden Diskussion an ein anderes Mitglied übertragen werden.
+5. Eine satzungsgemäß einberufene Mitgliederversammlung ist in jedem Fall beschlussfähig.
+6. Steht in Abstimmungen nur eine Option zur Wahl, so ist Abstimmung per Akklamation zulässig.
+
+
+## Satzungsänderungen
+
+1. Satzungsänderungen können von der Mitgliederversammlung mit einer Mehrheit von 2/3 der abgegebenen Stimmen beschlossen werden,
+    wenn dieser Punkt ausdrücklich bei der Einberufung auf der Tagesordnung genannt wurde.
+    Es ist eine Stimmabgabe per Fernkommunikationsmittel ab Zeitpunkt der Einladung zuzulassen.
+2. Satzungsänderungen, die von Aufsichts-, Berichts- oder Finanzbehörden aus formellen Gründen verlangt werden, kann der Vorstand von sich aus vornehmen.
+    Satzungsänderungen müssen den Mitgliedern alsbald in Textform mitgeteilt werden.
+3. Die Satzung besitzt Rechtsgültigkeit in ihrer Fassung in deutscher Sprache.
+    Zusätzlich ist eine englische Übersetzung anzufertigen,
+    die rein informativen Charakter besitzt.
+    Inhaltliche Änderungen sind immer an beiden Fassungen vorzunehmen.
+
+
+## Finanzprüfung
+
+1. Zwei Finanzprüfer*innen sind von der Mitgliederversammlung für jeweils ein Jahr zu wählen.
+    Diese haben die Aufgabe, das jeweils zurückliegende Geschäftsjahr des Vereins buchhalterisch zu prüfen.
+2. Den Finanzprüfer*innen sind zur Prüfung sämtliche Unterlagen des Vereins, Rechnungen, Bankauszüge und dergleichen zur Verfügung zu stellen.
+3. Finanzprüfer*innen müssen voll geschäftsfähige Mitglieder des Vereins sein.
+    Sie dürfen nicht Vorstandsmitglied sein.
+4. Die Kassenprüfung soll spätestens einen Monat vor der ordentlichen Mitgliederversammlung abgeschlossen sein.
+
+
+## Auflösung
+
+1. Die Auflösung des Vereins kann nur in einer Mitgliederversammlung,
+    die explizit zu diesem Zweck einberufen wurde,
+    mit einer Mehrheit von 3/4 der erschienenen Mitglieder aufgelöst werden.
```

Comment:
```suggestion
    mit einer Mehrheit von 3/4 der erschienenen Mitglieder und mit einer Anzahl an Gegenstimmen kleiner 5 aufgelöst werden.
```

Reply:
@p-snft Please have a look at the sentence, if it describes what you mean.

Reply:
I think that should be "oder" instead of "und". And the last "aufgelöst" should be "beschlossen". Sorry for not making that commitable changes. Couldn't figure out how.

Reply:
```suggestion
    beschlossen werden.
    Dabei gilt die Auflösung als beschlossen, wenn
    a) eine Mehrheit von 3/4 der abstimmenden Mitglieder diese befürwortet oder
    b) weniger als fünf Mitglieder gegen diese stimmen.
```

Reply:
Ich verstehe das "oder" an der Stelle nicht. Ich dachte es ist so, wenn von 40 Mitgliedern 30 den Verein auflösen wollen und 10 (also mehr als 5) dagegen stimmen, dann kann der Verein nicht aufgelöst werden. Oder hab ich das falsch verstanden? Dann müsste es doch "und" sein.

Reply:
Gemeint war, dass der Verein aufgelöst werden kann, wenn sich niemand mehr für diesen interessiert. Mindestens fünf Personen sollten Interesse an der Fortführung haben.

Reply:
```suggestion
    mit einer Mehrheit von 3/4 der abgegebenen Stimmen und maximal fünf Gegenstimmen aufgelöst werden.
```